cfme/fixtures/v2v.py

```diff
@@ -1,22 +1,18 @@
+import fauxfactory
 import pytest
+from riggerlib import recursive_update
+from widgetastic.utils import partial_match
 
 from cfme.fixtures.provider import setup_or_skip
-
-
-@pytest.fixture(scope="function")
-def migration_ui(appliance):
-    """Fixture to enable migration UI"""
-    appliance.enable_migration_ui()
-    yield
-    appliance.disable_migration_ui()
+from cfme.utils.generators import random_vm_name
 
 
 @pytest.fixture(scope='function')
-def v2v_providers(request, migration_ui, second_provider, provider):
+def v2v_providers(request, second_provider, provider):
     """ Fixture to setup providers """
     setup_or_skip(request, second_provider)
     setup_or_skip(request, provider)
-    yield second_provider, provider
+    yield provider, second_provider
```

Comment:
**SUGGESTION :** Why is this order changed? Initially all tests were designed to take second_provider(VMware),provider(RHV), It would be probably nice to keep it same than to re-order it in every test.

Reply:
I changed it because, I thought this is little-bit odd
```
@pytest.fixture(scope='function')
def host_creds(v2v_providers):
    """Add credentials to conversation host"""
    provider = v2v_providers[1]   <<<<<<<<
```
But anyway, changing index 0 to 1 is better than changing existing provider code.
Reverting it back, thanks.